src/bundle/Resources/public/js/scripts/sidebar/btn/location.edit.js

```diff
@@ -1,7 +1,32 @@
-(function () {
-    const editActions = document.querySelector('.ez-extra-actions--edit');
+(function (global, doc) {
+    const editActions = doc.querySelector('.ez-extra-actions--edit');
     const btns = [...editActions.querySelectorAll('.form-check [type="radio"]')];
     const form = editActions.querySelector('form');
+    const contentId = form.querySelector('#content_edit_content_info').value;
+    const checkDraftLink = global.Routing.generate('ezplatform.draft.has_no_conflict', { contentId });
+    const resetRadioButtons = () => btns.forEach(btn => btn.checked = false);
+    const addDraft = () => {
+        form.submit();
+        $('#draft-conflict-modal').modal('hide');
+    };
 
-    btns.forEach(btn => btn.addEventListener('change', () => form.submit(), false));
-})();
+    const changeHandler = () => {
+        const showModal = (modalHtml) => {
+            doc.querySelector('#draft-conflict-modal-wrapper').innerHTML = modalHtml;
+            doc.querySelector('#add-draft').addEventListener('click', addDraft, false);
```

Comment:
I'm not a fan of using IDs as a selector for JS. We more using classes for it. And also the naming is not proper. I would use class `.ez-btn--add-draft` or something like this. This might be applied in few places.